### PR TITLE
WM Class 4 - Pakize Bozkurt - SQL  Full-Stack-Project-Assessment

### PR DESCRIPTION
**Pull request by PakizeBozkurt**

### CODE REVIEW

File: client/src/companents/Main.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    const [videos, setVideos] = useState(dataVideos);
+    //console.log(videos);
+
+
+  return (
+    <div>
+      {videos.map(video =>(
+        <div>
+          <Videos videos={videos} data={video}/>
+          <DeleteBtn videos={setVideos} id={video.id} data={videos}/>
```
Comment (SallyMcGrath) [inline client/src/companents/Main.jsx:18]: What happens when you press this button?

File: client/src/companents/Main.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import DeleteBtn from './DeleteBtn';
+import Videos from './Videos';
+import dataVideos from "/home/codeyourfuture/Documents/SQL/W-2/Full-Stack-Project-Assessment/client/src/exampleresponse.json";
+
+
+
+function Main() {
+    const [videos, setVideos] = useState(dataVideos);
+    //console.log(videos);
```
Comment (SallyMcGrath) [inline client/src/companents/Main.jsx:10]: Please remove commented out code.
```suggestion
```

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Thanks for this @PakizeBozkurt. I can see you've put a lot of effort into this so far.

I think there's some work to do. I can't see your Lighthouse report in the files and I'm not sure all the requirements in Level 100 have been met. Take a look through and have a think about some of the questions I've raised. 

When you ran Lighthouse, what did the report say? What changes should you make? (I ran Lighthouse on your code.)

File: client/src/companents/Rating.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        variant="success"
+        onClick={() => handleIncrease()}
+      >
+        <ThumbUpIcon />
+      </button>
+      <span style={{ color: "darkgoldenrod", margin: "20px" }}>{rating}</span>
+      <button
+        style={{ margin: "20px" }}
+        variant="danger"
+        onClick={() => handleDicrease()}
```
[the remark this answers is not in the document]
Comment (PakizeBozkurt) [reply client/src/companents/Rating.jsx]: :+1:

File: client/src/companents/Videos.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+          </div>
+
+          <iframe
+            style={{ height: "500px", weihgt: "200px" }}
+            className="card-img-top"
+            alt={props.data.Title}
+            src={src}
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowFullScreen
+            title="Embedded youtube"
```
[the remark this answers is not in the document]
Comment (PakizeBozkurt) [reply client/src/companents/Videos.jsx]: title={props.videos.id}

File: client/src/companents/Main.jsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import DeleteBtn from './DeleteBtn';
+import Videos from './Videos';
+import dataVideos from "/home/codeyourfuture/Documents/SQL/W-2/Full-Stack-Project-Assessment/client/src/exampleresponse.json";
```
[the remark this answers is not in the document]
Comment (PakizeBozkurt) [reply client/src/companents/Main.jsx]: It was hardcode from json package. I am getting all data from database.

File: client/src/companents/Rating.jsx [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+      <button
+        style={{ margin: "20px" }}
+        variant="success"
+        onClick={() => handleIncrease()}
+      >
+        <ThumbUpIcon />
+      </button>
+      <span style={{ color: "darkgoldenrod", margin: "20px" }}>{rating}</span>
+      <button
+        style={{ margin: "20px" }}
```
[the remark this answers is not in the document]
Comment (PakizeBozkurt) [reply client/src/companents/Rating.jsx]: :smile_cat:  CSS :+1:

File: client/src/companents/Videos.jsx [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+        >
+          <div>
+            <h6 style={{ color: "#D43C31" }}>{props.data.title}</h6>
+            <span>
+              <Rating video={props.data} />
+            </span>
+          </div>
+
+          <iframe
+            style={{ height: "500px", weihgt: "200px" }}
```
[the remark this answers is not in the document]
Comment (PakizeBozkurt) [reply client/src/companents/Videos.jsx]: :+1:

File: client/src/companents/Main.jsx (hunk 2 above)
Comment (PakizeBozkurt) [reply client/src/companents/Main.jsx]: I removed them. :+1:

**Comment by PakizeBozkurt** [comment]
> Thanks for this @PakizeBozkurt. I can see you've put a lot of effort into this so far.
> 
> I think there's some work to do. I can't see your Lighthouse report in the files and I'm not sure all the requirements in Level 100 have been met. Take a look through and have a think about some of the questions I've raised.
> 
> When you ran Lighthouse, what did the report say? What changes should you make? (I ran Lighthouse on your code.)


Thank you so much @SallyMcGrath . 
I just did my best so far.

I ran Lighthouse on my code. I attached it in the bellow. :arrow_double_down: 
Screenshot from 2023-01-26 20-14-10

Have a fabulous week!

**Pull request closed without merging**